pkgs/by-name/la/latexminted/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "latexminted";
+  version = "0.1.0b1";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-O+p08Yg7seL3Nn4B+osV5EizLDfDTCh+YILRYgENPs4=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
```

```suggestion
  build-system = with python3Packages; [
```

According to recent studies...